integration_test/testutil_test.go

```diff
@@ -117,21 +117,20 @@ func lines(in []byte) []string {
 	return strings.Split(trimmed, "\n")
 }
 
-func (it *ITest) lookupExecutable(file string) error {
+func (it *ITest) LookupExecutable(file string) (string, error) {
```

Comment:
is there a particular reason to export this method? it's all in the same package.

Reply:
No particular reason. I found it more consistent with the other methods on `ITest`, which are all exposed when used outside of `testutil_test.go`.

Reply:
OK it's not a big issue.